Updating doc (#61)
JuliaSmoothOptimizers/BundleAdjustmentModels.jl: README.md

[ Info: The artifact venice/problem-1778-993923-pre.txt.bz2 was not found
```

> Special thanks to Célestine Angla for her initial work on this project as an internship.

Reviewer: ```suggestion
Special thanks to Célestine Angla for her initial work on this project during her internship.
```

> Special thanks to Célestine Angla for her initial work on this project as a master thesis.

Reviewer: ```suggestion
Special thanks to Célestine Angla for her initial work on this project as an internship.
```

Author: I changed it in latest commit.
